src/Abp/Authorization/Permission.cs

```diff
@@ -77,6 +100,7 @@ public class Permission
             Description = description;
             MultiTenancySides = multiTenancySides;
             FeatureDependency = featureDependency;
+            CustomProperties = customProperties;
```

Comment:
```suggestion
            CustomProperties = customProperties ?? new Dictionary<string, object>();
```

And remove checks in `public object this[string key]` get/set